### DISCUSSION TITLE
Jstree in pyang for generating html of yang

### DISCUSSION THREAD

**Discussion by Santhoshexp** [vakata/jstree#2617] [Q&A]
I am using jstree in pyang for generating html format of yang modules but the output is not generating range , description and other few parameters. Any update?

**Comment by vakata** [comment]
Sorry, no idea what you mean - my guess is you are using some UI tool build on top of jstree, which I can not help with.

**Reply by Santhoshexp** [reply]
No problem I changed the jstree code for my requirement thanks 👍
